src/components/widgets/TextWidget/TextWidget.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {WidgetTextMeta} from 'interfaces/widget'
+import {connect} from 'react-redux'
+import {Store} from '../../../interfaces/store'
+import styles from './TextWidget.less'
+
+const parse = require('html-react-parser')
+const marked = require('marked')
+
+interface TextWidgetOwnProps {
+    meta: WidgetTextMeta
+}
+
+interface TextWidgetProps extends TextWidgetOwnProps {
+    description: string
+}
+
+const TextWidget: React.FunctionComponent<TextWidgetProps> = (props) => {
+    const description = props.description
+    const htmlText = parse(marked(description))
```

Comment:
Will it break if description will be unparsable (non-valid html)? If so then we should check for this and provide a error message instead of page crash.